crypto/ct/ct_b64.c

```diff
@@ -39,7 +39,7 @@ static int ct_base64_decode(const char *in, unsigned char **out)
         goto err;
     }
 
-    outlen = EVP_DecodeBlock(outbuf, (unsigned char *)in, inlen);
+    outlen = EVP_DecodeBlock(NULL, outbuf, (unsigned char *)in, inlen);
```

Comment:
We can't change public APIs.  A new API is needed (EVP_DecodeBlock_ex).

Reply:
@paulidale Hi Pauli,
Would you please look at the EVP_DecodeBlock_ex() implementation when you get some bandwidth? I'm working on the test cases at this moment.

Thanks!